common/api/appui-react.api.md

```diff
@@ -3315,6 +3317,18 @@ export interface PanelSizeChangedEventArgs {
     size: number | undefined;
 }
 
+// @alpha
+export interface PanelsLayoutToolbarItem {
```

Comment:
`section` is already a required property of status ba

Reply:
Maybe it would make sense to add an empty object at least to other items: status bar item, backstage item. That would allow you to conditionally provide an item based on a layout (unless we decide to go with default locations for widgets, toolbar items).

Reply:
I was under the impression that the items returned by `get` methods were different than the one by `provide`, but yeah, if we intend to return the same ones, then we can't do that. And at the moment, I guess just like we removed the custom ones, we shouldnt add unused properties for now, so let's leave it as is, we can always change it later when we add Custom stuff management.